index.html

```diff
@@ -329,7 +329,13 @@ <h2>Best Practices and Recommendations</h2>
         
         <p>Checking if a tag is <a>valid</a> requires access to or a copy of the registry plus additional runtime logic. While content authors are advised to choose, generate, and exchange only valid values, language tag matching and other common language tag operations are designed so that validity checking is not needed. Features or functions that need to understand the specific semantic content of subtags are the main reason that a specification would normatively require <a>valid</a> tags as part of the protocol or document format.</p>
         
-        <p class="advisement" id="ltli-canonical-req"><a class="self" href="#ltli-canonical-req">&#x200B;</a>Content authors and implementations SHOULD use language tags that are <a>canonical Unicode locale identifiers</a> and SHOULD normalize language tags that they consume using the rules for producing canonical tags.</p>
+        <p class="advisement" id="ltli-canonical-author-req"><a class="self" href="#ltli-canonical-author-req">&#x200B;</a>Content authors SHOULD choose language tags that are <a>canonical Unicode locale identifiers</a>.</p>
+        
+        <p>The additional content restrictions and normalization steps found in <a href="https://www.unicode.org/reports/tr35/#Canonical_Unicode_Locale_Identifiers">Section 3</a> of [[LDML]] provide for better interoperability and consistency than that afforded by [[BCP47]] directly.</p>
```

Comment:
The restrictions include the conformance critera in section 3.3. of UTR 35, I assume? Here
https://www.unicode.org/reports/tr35/#BCP_47_Conformance
it says e.g. "No extlang subtags are allowed"
That means that widely existing tools need to be updated to conform ti LTLI, e.g. the W3C markup validator, see below screenshot:
 
![w3c-validator-extlang-check](https://user-images.githubusercontent.com/1177858/87390170-9698df00-c5a8-11ea-9935-03217994947f.PNG)

Do we really want that? I am still wondering whether this dependency on UTR 35 makes sense for Web content.

Reply:
UTR35's canonicalization would make `ar-abv` into `abv`--which BCP47 permits and even encourages. But I think you're probably right. Maybe the right thing is to apply the canonicalization rules when converting language tags to locales.

Where I'm trying to go with this is: HTML forms (and other similar specs) should format data values to be consistent with the context where the value is presented in the page. By this I mean "consistent with the language tag of the content". Formatting data values means using the language tag as a locale identifier, since locale-aware APIs are the ones that do the formatting.

Reply:
Understand, thanks for the explanation. Maybe the solution is, as you wrote here 
https://github.com/w3c/ltli/pull/12#discussion_r454419420
to make clearer what LTLI should take into account from UTR35.

Reply:
As promised in teleconference, I'm going to merge this now so others can see it. I think we have more to do here, so I'll create an issue to discuss.